src/lib/net/Cargo.toml

```diff
@@ -28,3 +29,4 @@ serde_derive = { workspace = true }
 rand = { workspace = true }
 uuid = { workspace = true, features = ["v4"] }
 async-trait = { workspace = true }
+bitreader = { workspace = true }
```

Comment:
Is this ever used?

Reply:
Nah, I thought I needed to be able to read the packed integers, but I ended up not needing it and I guess I forgot to remove the import.